.github/workflows/windows.yml

```diff
@@ -90,13 +117,24 @@ jobs:
       # For some reason WiX isn't currently on the GitHub runner path. This is a
       # temporary workaround until that is fixed.
       - name: Add WiX to Path
+        if: ${{ ! inputs.self-hosted-runner }}
         run: |
           "$env:WIX\bin" >> $env:GITHUB_PATH
+
+      # Configure tools in a self-hosted runner.
+      - name: Add python and wixtoolset to path
+        if: ${{ inputs.self-hosted-runner }}
+        run: |
+          "$env:LOCALAPPDATA\Programs\Python\Python312" >> $env:GITHUB_PATH
+          "C:\Program Files (x86)\WiX Toolset v3.11\bin" >> $env:GITHUB_PATH
+
       - name: Build (${{ inputs.profile }})
         run: |
           python mach build --use-crown --locked --${{ inputs.profile }}
           cp C:\a\servo\servo\target\cargo-timings C:\a\servo\servo\target\cargo-timings-windows -Recurse
       - name: Copy resources
+        # FIXME: what does this do?
+        if: ${{ ! inputs.self-hosted-runner }}
         run: cp D:\a\servo\servo\resources C:\a\servo\servo -Recurse
```

Comment:
On github runners checkouts or done on D: drive, but because it's space is limited, we build the servo on C drive (target folder on C drive), this looks like workaround for `mach` not being able to handle relocating target folder (not sure if it's still needed though).